assemble_synApps

```diff
@@ -584,7 +584,7 @@ if (exists($modules{MEASCOMP}))
 		
 		system("./configure --prefix=${PWD}");
 		system("make");
-		`make install || true`;
+		`sudo make install || true`;
```

Comment:
What is requiring root access here? The point of line 585 is that 'make install' will install everything into its own directory. I just ran the script on Ubuntu 22 and there were no root requirements.

Reply:
Edit: The build logs and artifacts are listed at the bottom of the summary of a build job, here: https://github.com/BAMresearch/EPICS-synApps-assemble/actions/runs/8023437425